sprite.go

```diff
@@ -1943,3 +1963,172 @@ func (p *SpriteImpl) checkAudioId() {
 		p.audioId = p.g.sounds.allocAudio()
 	}
 }
+
+// ------------------------ physic ----------------------------------------
+type PhysicsMode = int64
+
+const (
+	NoPhysic        PhysicsMode = 0 // Pure visual, no collision, best performance (current default) eg: decorators
+	KinematicPhysic PhysicsMode = 1 // Code-controlled movement with collision detection eg: player
+	DynamicPhysic   PhysicsMode = 2 // Affected by physics, automatic gravity and collision eg: items
+	StaticPhysic    PhysicsMode = 3 // Static immovable, but has collision, affects other objects : eg: walls
```

Comment:
对了，这里应该是 physics 而不是 physic？



```suggestion
	NoPhysics        PhysicsMode = 0 // Pure visual, no collision, best performance (current default) eg: decorators
	KinematicPhysics PhysicsMode = 1 // Code-controlled movement with collision detection eg: player
	DynamicPhysics   PhysicsMode = 2 // Affected by physics, automatic gravity and collision eg: items
	StaticPhysics    PhysicsMode = 3 // Static immovable, but has collision, affects other objects : eg: walls
```

Reply:
ok